syncer/syncer_test.go

```diff
@@ -55,7 +55,7 @@ type mockFetcher struct {
 	atxsError map[types.EpochID]error
 	atxsCalls uint32
 	tbsError  map[types.EpochID]error
-	tbsCalls  uint32
+	tbCalls   uint32
```

Comment:
If `tbsCalls` is renamed, should `tbsError` be renamed too?

Reply:
done. good call.